source/MaterialXTest/MaterialXRuntime/Runtime.cpp

```diff
@@ -2549,10 +2549,10 @@ TEST_CASE("Runtime: graph output connection", "[runtime]")
     mx::FileSearchPath searchPath(mx::FilePath::getCurrentPath() /
                                   mx::FilePath("libraries"));
     api->setSearchPath(searchPath);
-    api->loadLibrary(TARGETS_NAME, RuntimeGlobals::TARGETS_PATH());
-    api->loadLibrary(STDLIB_NAME, RuntimeGlobals::STDLIB_PATH());
-    api->loadLibrary(PBRLIB_NAME, RuntimeGlobals::PBRLIB_PATH());
-    api->loadLibrary(BXDFLIB_NAME, RuntimeGlobals::BXDFLIB_PATH());
+    api->loadLibraries(TARGETS_NAME, { RuntimeGlobals::TARGETS_PATH() });
```

Comment:
Would be good to change this to use a single call to "promote" that interface for this case as it's really not "safe" to load one at a time, especially if the order is incorrect.